npm/css-src/sass/_header.scss

```diff
@@ -1,8 +1,16 @@
 .govuk-header__tna-logo {
-    width: 33.33%;
+    width: 220px;
     float: left;
 }
 
 .govuk-header__tna-logo-image {
     width: 220px;
+}
+
+.govuk-header__tna-content {
```

Comment:
Where is the HTML element that this rule applies to? I can't find it, but I might be looking in the wrong place. Or is there a change to the templates that hasn't been checked in?

Reply:
This rule is applied to the HTML element on line 32 within `main.scala.html`

Reply:
Looking at https://github.com/nationalarchives/tdr-transfer-frontend/blob/position-logo/app/views/main.scala.html#L32 I can see `govuk-header__tna-logo` but not `.govuk-header__tna-logo-image` or `.govuk-header__tna-content`:

```
            <div class="govuk-header__tna-logo">
                <a href="/" class="govuk-header__link">
                    <span>
                        <image src="/assets/images/tna-horizontal-white-logo.svg" class="govuk-header__tna-logo-image"></image>
                    </span>
                </a>
            </div>

            <div class="govuk-header__content">
                <a href="#" class="govuk-header__link govuk-header__link--service-name">
                    @Messages("main.header")
                </a>
            </div>
```